src/mjolnir/CMakeLists.txt

```diff
@@ -25,12 +25,33 @@ if (UNIX)
   # Enables stricter compiler checks on a file-by-file basis
   # which allows us to migrate piecemeal
   set_source_files_properties(
+    bssbuilder.cc
+    complexrestrictionbuilder.cc
+    countryaccess.cc
+    dataquality.cc
+    directededgebuilder.cc
+    graphtilebuilder.cc
+    edgeinfobuilder.cc
+    elevationbuilder.cc
     graphenhancer.cc
+    graphvalidator.cc
     hierarchybuilder.cc
+    linkclassification.cc
     luatagtransform.cc
+    node_expander.cc
     osmdata.cc
+    osmpbfparser.cc
+    osmaccessrestriction.cc
     osmrestriction.cc
-    PROPERTIES COMPILE_FLAGS "-Werror -Wall")
+    osmway.cc
+    pbfadminparser.cc
+    restrictionbuilder.cc
+    shortcutbuilder.cc
+    timeparsing.cc
+    transitbuilder.cc
+    util.cc
+    validatetransit.cc
+    PROPERTIES COMPILE_FLAGS "-Werror -Wall -Wextra -Wno-error=missing-field-initializers")
```

Comment:
1. We have many lines like `OSMWay way{id}` that doesn't initialize all fields but the only one(id). That's why -Wno-error=missing-field-initializers is needed
2. gcc and clang have different set of -Wall so -Wextra is needed. E.g. clang's -Wextra contains `-Wsign-compare, -Wunused-parameter` cc @PureW